### PR TITLE
RC for the a percentile distances method

### PR DESCRIPTION
**Pull request by MartinLiebig**

### CODE REVIEW

File: src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		Percentile p = new Percentile();
+		p.setData(data);
+		lowerPercentile = p.evaluate(percentileThreshold * 100);
+		upperPercentile = p.evaluate((1 - percentileThreshold) * 100);
+	}
+
+	@Override
+	public double score(double value) {
+		if (value > upperPercentile && (scoringMode.equals(AnomalyUtilities.SCORING_MODE_BOTH) || scoringMode.equals(AnomalyUtilities.SCORING_MODE_ONLY_TOP))) {
+			return abs(value - upperPercentile);
```
Comment (darnu) [inline src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java:45]: Math.abs()

File: src/main/resources/anomalydetection/univariate_anomaly_detection.xml [hunk 2]
```diff
@@ -151,6 +175,64 @@
       </process>
 
 
+      </tutorialProcess>
+      <tutorialProcess key="process.operator_toolbox.outlier.filter_top_2_percent" title="Filter Top 2% of Titanic">
+      <description>
+        In this process we filter the titanic data set for all rows where either age or passenger fare are in the top 2% of the values
```
Comment (darnu) [inline src/main/resources/anomalydetection/univariate_anomaly_detection.xml:181]: Titanic

File: src/main/resources/anomalydetection/univariate_anomaly_detection.xml [hunk 3]
```diff
@@ -151,6 +175,64 @@
...
+
+        We choose only Age and Passenger Fare as attributes. We use percentile distance as a method and only consider top outliers with a threshold of 0.02. The scores are thus the distance to the 98-th percentile.
+        We choose maximum as an aggregation method for the final score. So any value above 0 in prediction as at least one value in the top 2% of the values. We use score in filter examples to remove them from the data set.
```
Comment (darnu) [inline src/main/resources/anomalydetection/univariate_anomaly_detection.xml:184]: "So any value above 0 in prediction as at least one value in the top 2% of the values."

I don't understand that sentence. you mean something like this:

_"So any value above 0 in the prediction belongs to the top 2%."_

File: src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
+package com.rapidminer.extension.anomalydetection.operator.univariate;
+
+import static java.lang.Math.abs;
+
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+
+import com.rapidminer.belt.column.Column;
+import com.rapidminer.belt.column.Statistics;
```
Comment (darnu) [inline src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java:8]: Remove unused import

**Review by darnu — CHANGES_REQUESTED** [review]

File: src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java (hunk 1 above)
Comment (MartinLiebig) [reply src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java:45]: done

File: src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java (hunk 4 above)
Comment (MartinLiebig) [reply src/main/java/com/rapidminer/extension/anomalydetection/operator/univariate/PercentileThresholdScorer.java:8]: done

File: src/main/resources/anomalydetection/univariate_anomaly_detection.xml (hunk 2 above)
Comment (MartinLiebig) [reply src/main/resources/anomalydetection/univariate_anomaly_detection.xml:181]: done

File: src/main/resources/anomalydetection/univariate_anomaly_detection.xml (hunk 3 above)
Comment (MartinLiebig) [reply src/main/resources/anomalydetection/univariate_anomaly_detection.xml:184]: done

**Pull request merged**